tests/IntrospectionTest.hack

```diff
@@ -8,6 +8,47 @@ final class IntrospectionTest extends PlaygroundTest {
     <<__Override>>
     public static function getTestCases(): this::TTestCases {
         return dict[
+            'validate interface introspection' => tuple(
+                '{
+                    __type(name: "IIntrospectionInterfaceA") {
+                        possibleTypes {
+                            name
+                        }
+                    }
+                 }',
+                dict[],
+                dict[
+                    '__type' => dict[
+                        'possibleTypes' => vec[
+                            dict[
+                                'name' => 'ImplementInterfaceB',
+                            ],
+                            dict[
+                                'name' => 'ImplementInterfaceC',
+                            ],
+                        ],
+                    ],
+                ],
+            ),
+            'validate interface introspection' => tuple(
```

Comment:
Should this test have a different name from the above test?

Reply:
copy 🍝